src/opencl_common.c

```diff
@@ -1499,7 +1500,8 @@ static void clear_profiling_events()
 	}
 }
 
-// Do the proper test using different global work sizes.
+// Do a test run with a specific global work size, return total duration
+// (or return zero for error or liomits exceeded)
```

Comment:
?

Reply:
That old comment would perhaps be proper for `opencl_find_best_gws()`. However, this comment is for `gws_test()` which tests a single GWS as (now) described. `opencl_find_best_gws()` calls `gws_test()` for each GWS.

Reply:
...and liomits is my stiff upper lip pronunciation of limits 🤣